apps/pom.xml

```diff
@@ -65,11 +65,7 @@
         <!-- The version of Java that we'll use for everything. -->
         <javaSourceVersion>21</javaSourceVersion>
 
-        <!-- Create a standard (but fake) timestamp for use in creating build artifacts so that
-            they are reproducible bit-by-bit no matter when/where they are generated.
-            See https://maven.apache.org/guides/mini/guide-reproducible-builds.html -->
-
-        <project.build.outputTimestamp>2024-01-10T17:15:29Z</project.build.outputTimestamp>
+        <project.build.outputTimestamp>2024-01-03T21:29:01Z</project.build.outputTimestamp>
```

Comment:
I suppose it probably doesnt matter _when_ this timestamp is, but why was this changed slightly?

Reply:
Dont think it matters. But i'll change it back as that wasn't in the AC. The AC was to remove the outdated comment.